cmake/target.cmake

```diff
@@ -61,6 +61,21 @@ if (CMAKE_CROSSCOMPILING)
         elseif (ARCH_S390X)
             set (ENABLE_GRPC OFF CACHE INTERNAL "")
             set (ENABLE_RUST OFF CACHE INTERNAL "")
+    elseif (ARCH_LOONGARCH64)
+            set (GLIBC_COMPATIBILITY OFF CACHE INTERNAL "")
+            set (ENABLE_LDAP OFF CACHE INTERNAL "")
+            set (OPENSSL_NO_ASM ON CACHE INTERNAL "")
+            set (ENABLE_JEMALLOC OFF CACHE INTERNAL "")
+            set (ENABLE_PARQUET OFF CACHE INTERNAL "")
+            set (ENABLE_GRPC OFF CACHE INTERNAL "")
+            set (ENABLE_HDFS OFF CACHE INTERNAL "")
+            set (ENABLE_MYSQL OFF CACHE INTERNAL "")
+            set (ENABLE_RUST OFF CACHE INTERNAL "")
+            set (ENABLE_LIBPQXX OFF CACHE INTERNAL "")
+            set (ENABLE_EMBEDDED_COMPILER OFF CACHE INTERNAL "")
+            set (ENABLE_DWARF_PARSER OFF CACHE INTERNAL "")
+            set (ENABLE_BLAKE3 OFF CACHE INTERNAL "")
+            set (COMPILER_CACHE disabled CACHE INTERNAL "")
```

Comment:
Disabling `COMPILER_CACHE` may negatively affect compilation times for LoongArch64 builds in CI. Could we remove l. 78?

Reply:
I have removed it.